How to use IPv6 in Docker using Proxmox and pfSense

bruj0 opened the pull request.

bruj0 commented:
Slightly updated version: https://github.com/bruj0/docker_ipv6_securely
I'll update it with the template once I have some feedback, thanks!

n-se commented:
What exactly should we review now? Your submitted tutorial or your link to the other tutorial?

If we should review https://github.com/bruj0/docker_ipv6_securely please update your pull request or submit a new pull request.

bruj0 commented:
I've updated and modified this commit to be better close to the template tutorial.
I'm also adding the disclaimer:

I have read and understood the Contributor's Certificate of Origin available at the end of 
https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md
and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: Rodrigo A. Diaz Leven <ramakandra@gmail.com>

To answer your question this PR is now ready for review.

n-se commented on tutorials/docker-ipv6-securely/01.en.md:
```diff
@@ -0,0 +1,208 @@
+We will need a way to provision VMs, for this article we selected Proxmox
+More information: 
+* https://www.proxmox.com/en/proxmox-ve/get-started
+* How to install it remotely on any server: https://www.linkedin.com/pulse/installing-any-os-headless-server-rodrigo-leven/
```
```suggestion
* How to install it remotely on any server: https://community.hetzner.com/tutorials/install-and-configure-proxmox_ve
```

n-se commented on tutorials/docker-ipv6-securely/01.en.md:
```diff
@@ -0,0 +1,208 @@
+* **vmbr0**: It will connect to the internet and receive both IPv4 and IPv6 traffic but will only have the main IPv4 assigned xxx.yyy.145.162/26
+* **vmbr1**: The internal network shared between VMs, 10.x.x.x
+* **vmbr2**: This bridge will hold the IPv6 network for our VMs, in this case a /64
+
```
Could you also show us ``/etc/network/interfaces`` for this configuration?

bruj0 replied:
Added the contents to the article.

n-se commented on tutorials/docker-ipv6-securely/01.en.md:
```diff
@@ -0,0 +1,208 @@
+tags: ["docker", "ipv6", "proxmox"]
+author: "Rodrigo A. Diaz Leven"
+author_link: "https://github.com/bruj0/"
+author_img: "https://avatars1.githubusercontent.com/u/2046474?s=460&v=4"
```
```suggestion
author_img: "https://avatars1.githubusercontent.com/u/2046474"
```
